pkg/haproxy/template.go

```diff
@@ -53,11 +53,10 @@ global
 defaults
 	log global
 
-	option http-server-close
-
-	# Disable logging of null connections (haproxy connections like checks).
-	# This avoids excessive logs from haproxy internals.
-	option dontlognull
+	# Defaults Options provided by user
```

Comment:
`# Defaults Options provided by user` these are not always provided by the user. Remove this comment. Instead add a link to HAProxy docs.